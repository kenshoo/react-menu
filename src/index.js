export { Menu } from "./components/menu";
