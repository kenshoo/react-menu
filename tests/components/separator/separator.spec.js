import React from "react";
import renderer from "react-test-renderer";

import { Separator } from "../../../src/";

describe("Separator component", () => {
  test("renders children", () => {
    const cmp = renderer.create(<Separator>TEST</Separator>);
    expect(cmp).toMatchSnapshot();
  });

  test("recieves className", () => {
    const cmp = renderer.create(
      <Separator className="TEST_CLASS">TEST</Separator>
    );
    expect(cmp).toMatchSnapshot();
  });
});
