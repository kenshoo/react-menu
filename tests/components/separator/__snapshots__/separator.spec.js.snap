// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator component recieves className 1`] = `
<div
  className="sc-gZMcBi cwfHym TEST_CLASS"
>
  TEST
</div>
`;

exports[`Separator component renders children 1`] = `
<div
  className="sc-gZMcBi cwfHym"
>
  TEST
</div>
`;
